Bushra-hw1/p1_fork_demo.c

```diff
@@ -0,0 +1,30 @@
+// C++ Code to demonstrate getppid() 
+#include <stdio.h> 
+#include <unistd.h> 
+  
+// Driver Code 
+int main() 
+{ 
+    pid_t pid1, pid11_new,pid12_new;
+    printf("Current Process id: %d \n", getpid()); 
+    printf(" Current Parent Process id: %d \n", getppid()); 
+
+    pid1 = fork();
+
+    if (pid1== 0) {    
+        printf("1st Child Process id: %d \n", getpid()); 
+        printf("Parent Process id: %d \n", getppid()); 
+        pid11_new = fork();
+        if (pid11_new == 0){
+            printf("11nd Child Process id: %d \n", getpid()); 
+            printf("Parent Process id: %d \n", getppid()); 
+pid12_new = fork();
```

Comment:
Here creating 3rd process in frame of 2nd process. Try here wait 2nd process finish and after that create 3rd process.